test/auto_parallel/CMakeLists.txt

```diff
@@ -74,6 +74,9 @@ if(WITH_DISTRIBUTE AND WITH_GPU)
   py_test_modules(test_auto_tuner_compare MODULES test_auto_tuner_compare)
   set_tests_properties(test_auto_tuner_compare
                        PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE" TIMEOUT 100)
+  py_test_modules(test_pass_quantization MODULES test_pass_quantization)
+  set_tests_properties(test_pass_quantization
+                       PROPERTIES LABELS "RUN_TYPE=EXECLUSIVE" TIMEOUT 60)
```

Comment:
Why `EXECLUSIVE`?

Reply:
Just a name, the reason for naming cannot be confirmed anymore

Reply:
If there are good suggestions, they can be changed